src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -12,7 +14,22 @@
     <t:base/>
     <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
     <div class="srp-container">
-      <div class="gcse-searchresults-only"></div>
+      <div class="srp-child">
+        <div class="gcse-searchresults-only"></div>
+      </div>
+      <div class="srp-child">
+        <%ArrayList<Card> cards = (ArrayList<Card>)request.getAttribute("card_list");%>
+          <div class="card">
+            <div class="card-content">
+              <a href="<%=cards.get(0).getLink()%>" title="card-source" target="_blank">
+                <span class="card-title"><%=cards.get(0).getTitle()%></span>
+              </a>
+              <p><%=cards.get(0).getCode()%></p>
+              <span class="card-title">Description</span>
+              <p><%=cards.get(0).getDescription()%></p>
+            </div>
+          </div>
+      </div>
```

Comment:
does these line only used to display one card? if so, TODO for adding another two cards.

Reply:
yeah, I wanted to check if ya'll are ok with how it looks so far as the other two will follow a similar format.